projects/addon-mobile/internal/primitive-calendar-mobile/primitive-calendar-mobile.style.less

```diff
@@ -110,13 +110,13 @@
 }
 
 @media screen and (max-width: 22.4375rem) {
-    .cell,
-    :host._ios .cell {
+    .t-cell,
+    :host._ios .t-cell {
```

Comment:
Please also check host binging classes. Might be in a form of:
```
    host: {
        '[class._ios]': 'isIOS',
    },
```

Reply:
We discussed this and decided not to touch these classes.